packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -1177,7 +1183,124 @@ export class TeamsAppSolution implements Solution {
 
   @hooks([ErrorHandlerMW])
   async checkPermission(ctx: SolutionContext): Promise<Result<any, FxError>> {
-    return ok(Void);
+    const canCheckPermission = this.checkWhetherSolutionIsIdle();
+    if (canCheckPermission.isErr()) {
+      return canCheckPermission;
+    }
+
+    const provisioned = this.checkWetherProvisionSucceeded(ctx.config);
```

Comment:
we still check azure permission?

Reply:
We still need to check whether it has provisioned, because only after provision Teams App and AAD would be created